tests/test_marker_definitions.py

```diff
@@ -9,7 +11,7 @@
 )
 from portabletext_html.types import Block, Span
 
-sample_texts = ['test', None, 1, 2.2, '!"#$%&/()']
+sample_texts = ['test', None, 1, 2.2, '!"#$%/()']
```

Comment:
any reason for changing this?

Reply:
No, this was something I was testing. Rolling back for now